core/templates/dev/head/components/alert-message/alert-message.directive.js

```diff
@@ -51,3 +51,4 @@ oppia.directive('alertMessage', [function() {
     }
   };
 }]);
+    
```

Comment:
Missing newline at end. Should be a lint check.

Reply:
Oh, I turned off the linter for this very commit, Done. Thanks!

Reply:
I think that needs investigation. Create a set of minimal repro steps and file an issue?

Reply:
Oh, Ok I'll try to file the issue ASAP after double checking the Travis 
(it successfully checks on local).